stdlib/public/core/StringRangeReplaceableCollection.swift

```diff
@@ -267,7 +266,6 @@ extension String : StringProtocol, RangeReplaceableCollection {
   ///
   /// - Parameter i: A valid index of the string. `i` must be less than the
   ///   string's end index.
-  @inlinable // FIXME(sil-serialize-all)
   public subscript(i: Index) -> Character {
```

Comment:
If this gives acceptable perf for ascii strings then that's excellent. Otherwise we may have to split off some fast-paths for these.

Reply:
The benchmarks didn't show any regressions (I did already run them locally). I also tested with a dedicated micro benchmark for subscript. So I don't think it's a problem.